bindings/nodejs/lib/wallet/account.ts

```diff
@@ -789,14 +789,14 @@ export class Account {
      * or custom inputs.
      * @returns The minting transaction and the token ID.
      */
-    async prepareMintNativeToken(
-        params: MintNativeTokenParams,
+    async prepareCreateNativeToken(
```

Comment:
Also update the documentation so it's different from the mint one?

Reply:
Done